azurepipelines/adu-yocto-build.yml

```diff
@@ -15,84 +15,68 @@
 #                      or 'deliveryoptimization-agent' for 1.0.0 or later.
 # 
 parameters:
-- name: build_env
-  displayName: Build environment
-  type: string
-  default: azure-iot-hub-device-update-develop
-  values:
-  - azure-iot-hub-device-update-develop
-  - azure-iot-hub-device-update-1-0-0
+
 - name: yocto_release
   displayName: Yocto release 'version'
   type: string
-  default: honister
+  default: kirkstone
   values:
-  - honister
-  - warrior
-- name: meta_layer_branch
-  displayName: Meta layers 'branch'
-  type: string
-  default: honister
-  values:
-  - honister
+  - kirkstone
+
 - name: adu_src_uri
   displayName: ADU URI
   type: string
-  default: "gitsm://github.com/azure/iot-hub-device-update"
+  default: "git://github.com/Azure/iot-hub-device-update" 
+
 - name: adu_git_branch
   displayName: ADU Git branch Name
   type: string
   default: develop
   values:
-    - main
     - develop
+
 - name: adu_git_commit
   displayName: ADU Git commit 
   type: string
-  default: e099aaca0ad2b8849de24a13b1aa8e3ddf009251
+  default: 60bb98ae3631419b393c528f7dc3cf0797b231e6
```

Comment:
should it be latest commit which is [10791ac6bd98eaf4e87ea54350d3e9c8a2cec1da ](https://github.com/Azure/iot-hub-device-update/commit/10791ac6bd98eaf4e87ea54350d3e9c8a2cec1da) now?

Reply:
pls double check

Reply:
These will have to be updated in a release.